lib/elixir_bot.ex

```diff
@@ -2,19 +2,64 @@ defmodule ElixirBot do
   use Slacker
   use Slacker.Matcher
 
-  match ~r/.*shots fired.*/, :hate_on
-  match ~r/tell which language is the best/, :best_lang
-  match ~r/.*najlepszy jezyk.*/, :best_lang
-  match ~r/.*dowcip.*/, :joke
-  match ~r/mow do tostera/, :gobot
-  match ~r/.*please.*/, :plz
-  match ~r/dodaj: (w+)/, :add_list
-  match ~r/.*pokaz liste.*/, :show_list
-  match ~r/.*resetuj liste.*/, :reset_list
+  match ~r/t-001.*shots fired.*/, :hate_on
+  match ~r/t-001.*najlepszy jezyk.*/, :best_lang
+  match ~r/t-001.*dowcip|smieszkuj.*/, :joke
+  match ~r/t-001.*please.*/, :plz
+  match ~r/t-001.*dodaj (\w+) (.*)/, :add_list
+  match ~r/t-001.*pokaz liste.*/, :show_list
+  match ~r/t-001.*resetuj|wyczysc liste.*/, :reset_list
+  match ~r/t-001.*pomocy.*/, :help
+  match ~r/t-001.*wysylaj|wysyloj|koniec dnia dziecka.*/, :list_ready
 
   def start(_type, _args) do
-    {:ok, tars} = ElixirBot.start_link("token")
-    {:ok, pid}  = ShoppingList.start_link
+    {:ok, listener} = Task.start_link(fn -> loop() end)
+    Process.register(listener, :listener)
+    {:ok, pid} = ShoppingList.start_link
```

Comment:
Use Supervisor with a GenServer or even with an Agent which is specialized GenServer for holding state.